docs/products/kafka/howto/connect-java.rst

```diff
@@ -0,0 +1,178 @@
+Java samples for testing Aiven for Apache Kafka®
+=================================================
+
+The sample Java code in this article demonstrates how to connect to your Aiven for Apache Kafka service and send messages into a topic.
+
+.. note::
+
+    The examples in this article provide two different options for authentication: SSL and SASL-SSL. For more information on these  authentication methods read :doc:`our article on Kafka authentication types <../concepts/auth-types>`.
+
+.. note::
+
+    The instructions below assume that there is existing topic with name ``demo-topic`` and that you have created the keystore ``client.keystore.p12`` and truststore ``client.truststore.jks``. For details, see :doc:`our article on configuring Java SSL to access Kafka <../howto/keystore-truststore>`.
+
+Relying on SSL authentication
+-----------------------------
+
+Create and run a producer, that sends a message into the topic ``demo-topic``:
+
+.. code::
+
+         package io.aiven.kafkaExample;
+         import java.io.IOException;
+         import java.util.Properties;
+         import org.apache.kafka.clients.producer.KafkaProducer;
+         import org.apache.kafka.clients.producer.ProducerRecord;
+
+         public class Producer {
+             public static void main(String[] args) throws IOException {
+                 Properties props = new Properties();
+                 props.put("bootstrap.servers", "service-uri.aivencloud.com:port");
```

Comment:
Can we reference variables instead of strings?

Reply:
done!